components/RichText/RichText.tsx

```diff
@@ -0,0 +1,177 @@
+  textColor = 'black',
+  textAlign = 'left',
+  className,
+}: RichTextProps) => {
+  const printColor = 'print:text-black';
+
+  const rendered = render(wysiwyg, {
+    markResolvers: {
+      [MARK_STYLED]: (children, props) => {
+        // Custom classes are string of class name(s) separated by spaces, e.g., 'ood-has-tab-before su-sans su-before-bg-bay-dark', 'su-bold'
+        const { class: wysiwygCustomClasses = '' } = props;
+
+        // Expanded icon options as needed
+        const hasIcon = wysiwygCustomClasses.includes('su-link--action');
+
+        /**
+         * Split the string of legacy classes into an array, filter out empty strings, and map to the corresponding TW class names
+         * If a match is not found in the wysiwygClasses object, return the original legacy custom class name from the WYSIWYG
+         * Stitch the array back into a string at the end
+         */
+        const finalClasses = wysiwygCustomClasses.split(/\s+/)
+          .filter(Boolean)
+          .map((className) =>
+            className in wysiwygClasses ? wysiwygClasses[className as WysiwygClassesType] : className,
+          ).join(' ');
+
+        return (
+          <Text
+            as="span"
+            icon={hasIcon ? 'chevron-right' : undefined}
+            iconProps={{ className: hasIcon ? 'inline-block ml-03em group-hocus-within:translate-x-02em' : undefined }}
+            className={finalClasses}
+          >
+            {children}
+          </Text>
+        );
+      },
+      [MARK_BOLD]: (children) => <strong>{children}</strong>,
+      [MARK_ITALIC]: (children) => <em>{children}</em>,
+      [MARK_STRIKE]: (children) => <del>{children}</del>,
+      [MARK_LINK]: (children, props) => {
+        const {
+          href,
+          target,
+          linktype,
+          anchor,
+          custom,
+        } = props;
+
+        // Prevents empty links in the WYSIWYG editor from rendering (a11y issue)
+        if (!children || (typeof children === 'string' && !children.trim())) {
+          return null;
+        }
+        /**
+         * The data shape of the inline links in WYSIWYG is different form regular Storyblok link field.
+         * Here we structure it to match the sbLink type so we can pass that into SbLink component.
+         */
+        const sbLink = {
+          linktype,
+          cached_url: linktype === 'story' ? href : '',
+          url: linktype === 'asset' || linktype === 'url' ? href : '',
+          email: linktype === 'email' ? href : '',
+          anchor: linktype === 'story' ? anchor : '',
+          // The WYSIWYG inline links automatically add a target="_self" by default which is unnecessary
+          target: target === '_blank' ? '_blank' : undefined,
+        };
+
+        return (
+          <SbLink
+            link={sbLink}
+            classes="group"
+            attributes={custom} // Custom link attributes
+          >
+            {children}
+          </SbLink>
```

Comment:
Might refactor `SbLink` to become the `CtaLink` like in Momentum/Tour/Emergency later so everything's the same